sample-node-config/gcp/docker-compose.yaml

```diff
@@ -61,9 +90,9 @@ services:
     volumes:
     - ./prometheus.yml:/etc/prometheus/prometheus.yml:ro
 
-  promtail:
-    image: grafana/promtail:1.4.1
-    volumes:
-      - /var/lib/docker/containers:/var/lib/docker/containers
-      - ./promtail-config.yml:/etc/promtail/promtail-config.yml
-    command: -config.file=/etc/promtail/promtail-config.yml
+  # promtail:
+  #   image: grafana/promtail:1.4.1
+  #   volumes:
+  #     - /var/lib/docker/containers:/var/lib/docker/containers
+  #     - ./promtail-config.yml:/etc/promtail/promtail-config.yml
+  #   command: -config.file=/etc/promtail/promtail-config.yml
```

Comment:
Commented code

Reply:
Good catch. Will delete it as I don't know enough about promtail